x/concentrated-liquidity/incentives_test.go

```diff
@@ -3535,6 +3527,81 @@ func (s *KeeperTestSuite) TestQueryAndClaimAllIncentives() {
 	}
 }
 
+func (s *KeeperTestSuite) TestClaimAllIncentivesForPosition() {
+	testCases := []struct {
+		name             string
+		blockTimeElapsed time.Duration
+		expectedCoins    sdk.Coins
+		expectedForfeit  int
+		expectedErr      bool
+	}{
+		{
+			name:             "Claim with same blocktime",
+			blockTimeElapsed: 0,
+			expectedCoins:    sdk.NewCoins(),
+			expectedForfeit:  0,
```

Comment:
Would it be possible to check nonzero forfeited amounts as well?

Reply:
i wrote an explanation here about this: https://github.com/osmosis-labs/osmosis/pull/5290#discussion_r1209548349

also the main difference between this test and `TestQueryAndCollectIncentives` is that this test specifically only focuses on `updateUptimeAccumulatorsToNow` which was added after `TestQueryAndCollectIncentives`.  So if you remove `updateUptimeAccumulatorsToNow` and run `TestQueryAndCollectIncentives` it'll pass whereas this test will fail

Reply:
I see, thanks for the clarification!